Microsoft.VisualStudio.CodeTools/CodeToolsSetup/buildMSM.xml

```diff
@@ -55,11 +55,16 @@
     <Platform>Mixed Platforms</Platform>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(VisualStudioVersion)'==''">
+    <VisualStudioVersion Condition="'$(MSBuildToolsVersion)'=='14.0'">14.0</VisualStudioVersion>
+    <VisualStudioVersion Condition="'$(MSBuildToolsVersion)'!='14.0'">12.0</VisualStudioVersion>
```

Comment:
It does. The rest of the build today uses Visual Studio 2013, so I updated this line to match. I can include a third line if you think that would be better:

```xml
<VisualStudioVersion Condition="'$(MSBuildToolsVersion)'=='14.0'">14.0</VisualStudioVersion>
<VisualStudioVersion Condition="'$(MSBuildToolsVersion)'=='12.0'">12.0</VisualStudioVersion>
<VisualStudioVersion Condition="'$(VisualStudioVersion)'==''">11.0</VisualStudioVersion>
```